packages/server/src/resolvers/Mutation.ts

```diff
@@ -1,9 +1,14 @@
 import { MutationResolvers } from "../schema/__generated__/graphql";
+import { sequelize } from "../store";
 
 export const Mutation: MutationResolvers = {
   resetDocker: async (parent, args, { docker }) => {
     return await docker.reset();
   },
+  resetVolume: async (parent, args, { docker, dataSources }) => {
```

Comment:
See automated review on this line: https://github.com/wpilibsuite/Axon/pull/216/checks?check_run_id=2943497452